### PR TITLE
closes #63

### LINKED ISSUE #63: Add a comprehensive example for FileConnector
The tests are a good example but it would be better for new users if they see an explicit examples module

### PR DESCRIPTION
**Pull request by ASRagab**
Let me know if you had another idea in terms of where to add this example, including in code

### CODE REVIEW

File: docs/quickstart/index.md [hunk 1]
```diff
@@ -7,4 +7,98 @@ FileConnector
...
+import java.io.{ByteArrayInputStream, File, IOException}
+import java.nio.charset.StandardCharsets
+import java.nio.file.Path
+
+import scala.language.postfixOps
+import java.nio.file.Files
+import java.nio.file.Paths
+
+object FileConnectorDemo extends ZIOAppDefault {
+  val path = Files.createTempDirectory("zio-connect-file")
```
Comment (adrianfilip) [inline docs/quickstart/index.md:29]: We don't need to use zio-nio. This should also be done with FileConnector.

Comment (ASRagab) [reply docs/quickstart/index.md:29]: this is using `java.nio` and I guess I wanted to show that you can interact with regular java file apis as well.

**Comment by adrianfilip** [comment]
The examples look great. I would move it to an examples module though so users can run it and play with it. (ex like in https://github.com/zio/zio-aws for instance where there is a separate examples module).

File: docs/quickstart/index.md (hunk 1 above)
Comment (adrianfilip) [reply docs/quickstart/index.md:29]: Let's change it please. 🙏 
We want to show you can do everything without moving away from the FileConnector api

**Comment by ASRagab** [comment]
> @ASRagab The examples look great. I would move it to an examples module though so users can run it and play with it. (ex like in https://github.com/zio/zio-aws for instance where there is a separate examples module).

@adrianfilip updated moved to `examples` sub project, can add additional sub projects easily when new connectors come on--line

**Review by adrianfilip — APPROVED** [review]

**Pull request merged**